tests/core/pm-module/test_pm_init.py

```diff
@@ -14,11 +20,23 @@ def web3():
         from web3.pm import PM
```

Comment:
if `ethpm` is a strict dependency of web3 then is these try except block needed?

Reply:
@voith We've decided to continue with `ethpm` development on a separate branch `pm-api` until we reach a stable v1. So this strict dependency will only exist on the experimental `pm-api` branch - and once v1 is reached we will ensure all dependency issues are stable before merging `pm-api` into `master`

Reply:
You're right.  Nick and I decided to just delete the `ethpm` requirement from `setup.py`.  That way, we can still possibly later merge the `pm-api` branch into master and have the web3 PM module be optional alpha functionality.  Thanks for noticing this discrepancy.